tests/rustls_version.rs

```diff
@@ -0,0 +1,70 @@
+use std::fs::File;
+use std::io::Read;
+use std::path::PathBuf;
+use std::{slice, str};
+
+use toml::Table;
+
+use rustls_ffi::rustls_version;
+
+/// Test that the output of `rustls_version()` matches what we expect based on Cargo.toml state.
+///
+/// In particular this ensures that the Rustls version reported in the rustls-ffi version string
+/// matches the version of the Rustls dependency that rustls-ffi was built with.
+///
+/// If this test starts to fail, you probably forgot to update `RUSTLS_CRATE_VERSION` in
+/// `build.rs`.
+#[cfg_attr(miri, ignore)] // Requires file I/O
+#[test]
+fn rustls_version_match() {
+    // Parse Cargo.toml as a generic TOML Table.
```

Comment:
I think this is a bit nicer than what I would have done, which is to shell out to `cargo metadata` and then squish the output through jq!

Reply:
I have to re-learn/forget `jq` syntax every time I reach for it!